README.md

```diff
@@ -70,14 +70,10 @@ To build Clara Genomics Analysis -
 ```bash
 mkdir build
 cd build
-cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=install -Dcga_cuda_gen_all_arch=OFF
+cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=install
```

Comment:
if decided to remove -Dcga_cuda_gen_all_arch=OFF altogether for cudapoa, cudaaligner as well as cudamapper, then README documentation needs to be updated as well.